### DISCUSSION TITLE
Great Project

### DISCUSSION THREAD

**Discussion by elral** [frank26080115/alpha-fairy#25] [General]
After a longer time I bought a new camera (Sony Alpha 7 IV) to start making pictures again.
While looking around the Internet for accessories and also for remote controls I found less informations and only ready to use remote controls.
At least I did a search on Github as I am also interested in electronics and having some reposorities on github, and I found this project.
I just want to say that I am impressed what about this project and I directly ordered a M5StickC. Once I will get the M5Stick I will give it a try. Until there is a lot if documentation available to read, also excellent!

As I do not really like the Arduino IDE most of my projects are set up with PlatformIO which I like much more. I did the same for this code which took me some hours, but in the end it's compiling fine without any warnings.

Best regards

**Comment by tao-j** [comment]
Hey, I have made it work with PIO #28 , you may want to check it out.

**Reply by elral** [reply]
I already did it ;)
